TombEngine/Objects/TR2/Entity/tr2_dragon.cpp

```diff
@@ -27,6 +29,7 @@ namespace TEN::Entities::Creatures::TR2
 	const auto DragonMouthBite = BiteInfo(Vector3(35.0f, 171.0f, 1168.0f), 12);
 	const auto DragonSwipeAttackJointsLeft  = std::vector<unsigned int>{ 24, 25, 26, 27, 28, 29, 30 };
 	const auto DragonSwipeAttackJointsRight = std::vector<unsigned int>{ 1, 2, 3, 4, 5, 6, 7 };
+	const auto DragonBackSpineJoints = std::vector<unsigned int>{ 21, 22, 23 };
 
```

Comment:
Meshes 21, 22 and 23 are the Dragon lower jaw and tongue. Maybe this was meant to be called DragonLowerJawJoints?

Reply:
No it's the spine joints in the dragon back, OG remove it from the code to avoid it being seen in-game